index.js

```diff
@@ -254,7 +290,9 @@ function fileListing(mask, pageTemplate, route, req, res) {
 		const o = {
 			name: path.relative(config.imagePath, f),
 			size: stat.size,
-			mtime: stat.mtime
+			mtime: stat.mtime,
+			mtimeSave: stat.mtime.toString(),
+			c: (pageTemplate == "links" ? fs.readFileSync(`${f}`, { encoding: "utf8" }).trim() : undefined) /* undefined is not saved into JSON */
```

Comment:
Just making a note I should probably check if file size from stats is less than 1024 here. Just in case there is large file that is not a link.